torchmetrics/functional/text/rouge.py

```diff
@@ -240,7 +269,8 @@ def _rouge_score_compute(sentence_results: Dict[str, List[Tensor]]) -> Dict[str,
 
 def rouge_score(
     preds: Union[str, List[str]],
-    targets: Union[str, List[str]],
+    targets: Union[str, List[str], List[List[str]]],
```

Comment:
Please, update a docstring. If `targets` are allowed to be of a type `List[List[str]]`, we actually pass in the iterable of iterables